src/components/ec-blog-item/ec-blog-item.vue

```diff
@@ -0,0 +1,26 @@
+<template>
+  <div>
+    <p>{{ title }}</p>
+    <p>{{ shortDescription }}</p>
+    <p>{{ author }}</p>
+    <p>{{ category }}</p> hello
+  </div>
+</template>
+
+<script>
+export default {
+  name: 'EcBlogItem',
+  props: {
+    title: { type: String },
+    shortDescription: { type: String },
+    author: { type: String },
+    category: { type: String },
+  },
+};
+</script>
+<style>
+p {
```

Comment:
Setting the `p` style in this component will affect paragraphs in other components too. You should either:
1. use [scoped styles](https://vue-loader.vuejs.org/guide/scoped-css.html#deep-selectors) (not recommended in this project but can be used in other vue projects)
2. Use CSS classes and BEM - that's what all other components do in this project

```html
<div class="ec-blog-item">
  <p class="ec-blog-item__title">...</p>
  <p class="ec-blog-item__short-description">...</p>
  ...
```
```css
.ec-blog-item {
  &__title,
  &__short-description {
    ... your css here.
  }
}
```